dipy/tracking/local_tracking.py

```diff
@@ -248,8 +253,12 @@ def __init__(self, direction_getter, stopping_criterion, seeds, affine,
         if particle_count <= 0:
             raise ValueError("The particle count must be greater than 0.")
 
-        self.directions = np.empty((maxlen + 1, 3), dtype=float)
+        if min_wm_pve_before_stopping < 0 or min_wm_pve_before_stopping > 1:
```

Comment:
`if not 0 < min_wm_pve_before_stopping < 1` instead

Reply:
done.